src/ui/public/chrome/chrome.html

```diff
@@ -1,71 +1,48 @@
 <div class="content" chrome-context >
   <!-- TODO: These config dropdowns shouldn't be hard coded -->
-  <nav class="app-links-wrapper" ng-show="chrome.getVisible()">
```

Comment:
Not a big deal, but in my opinion, if we can keep markup tags on a single line, we should